pkgs/development/node-packages/default-v10.nix

```diff
@@ -35,6 +35,10 @@ nodePackages // {
     name = "bitwarden-cli-${drv.version}";
   });
 
+  insect = nodePackages.insect.override {
+    nativeBuildInputs = [ pkgs.psc-package pkgs.purescript nodePackages.pulp ];
```

Comment:
I'm not sure if node2nix sets up any default `nativeBuildInputs` but just in case this really should be

```nix
  insect = nodePackages.insect.override (drv: {
    nativeBuildInputs = drv.nativeBuildInputs or [] ++ [ pkgs.psc-package pkgs.purescript nodePackages.pulp ];
  });
```

Reply:
seems like it doesnt https://github.com/teto/nixpkgs/blob/dece5e6d0bd5556018ee1faf19bf94f3e1672849/pkgs/development/node-packages/node-env.nix#L376

Reply:
Ok, but we don't want to step on its toes if `buildNodePackage` is ever updated in the future to start using `nativeBuildInputs`.